src/lib.rs

```diff
@@ -82,9 +111,9 @@ impl Client {
         }
     }
 
-    pub async fn fetch(&self, queues: &[String]) -> Result<Option<Job>> {
+    pub async fn fetch(&self, queues: Vec<Cow<'static, str>>) -> Result<Option<Job>> {
```

Comment:
Because we have to pass the queues to the execution channel, so we can't have a `&'a str`, and forcing the user to only have `&'static str` queues seems a problem.